scraping/x/adajobo_scrapper.py

```diff
@@ -0,0 +1,372 @@
+            content_size_bytes=1072,
+        ),
+    ]
+
+    results = await scraper.validate(entities=true_entities)
+    print(f"Validation results: {results}")
+
+
+async def test_multi_thread_validate():
+    scraper = AdajoboTwitterScraper()
+
+    true_entities = [
+        DataEntity(
+            uri="https://twitter.com/bittensor_alert/status/1748585332935622672",
+            datetime=dt.datetime(2024, 1, 20, 5, 56, tzinfo=dt.timezone.utc),
+            source=DataSource.X,
+            label=DataLabel(value="#Bittensor"),
+            content='{"username":"@bittensor_alert","text":"🚨 #Bittensor Alert: 500 $TAO ($122,655) deposited into #MEXC","url":"https://twitter.com/bittensor_alert/status/1748585332935622672","timestamp":"2024-01-20T5:56:00Z","tweet_hashtags":["#Bittensor", "#TAO", "#MEXC"]}',
+            content_size_bytes=318,
+        ),
+        DataEntity(
+            uri="https://twitter.com/HadsonNery/status/1752011223330124021",
+            datetime=dt.datetime(2024, 1, 29, 16, 50, tzinfo=dt.timezone.utc),
+            source=DataSource.X,
+            label=DataLabel(value="#faleitoleve"),
+            content='{"username":"@HadsonNery","text":"Se ele fosse brabo mesmo e eu estaria aqui defendendo ele, pq ele não foi direto no Davi já que a intenção dele era fazer o Davi comprar o barulho dela 🤷🏻\u200d♂️ MC fofoqueiro foi macetado pela CUNHÃ #faleitoleve","url":"https://twitter.com/HadsonNery/status/1752011223330124021","timestamp":"2024-01-29T16:50:00Z","tweet_hashtags":["#faleitoleve"]}',
+            content_size_bytes=492,
+        ),
+    ]
+
+    def sync_validate(entities: list[DataEntity]) -> None:
+        """Synchronous version of eval_miner."""
+        asyncio.run(scraper.validate(entities))
+
+    threads = [
+        threading.Thread(target=sync_validate, args=(true_entities,)) for _ in range(5)
+    ]
+
+    for thread in threads:
+        thread.start()
+
+    for t in threads:
+        t.join(120)
+
+
+if __name__ == "__main__":
+    bt.logging.set_trace(True)
+    asyncio.run(test_multi_thread_validate())
```

Comment:
Just to confirm, when you run these, are all the validation results successful? If it's not too much work, could you paste the output in the PR description?